app/services/markdown_processor/fixer/base.rb

```diff
@@ -0,0 +1,95 @@
+module MarkdownProcessor
```

Comment:
We could probably make many of these methods here private...but I like that we explicitly test them. I'm open to suggestions about this!